poetry.lock

```diff
@@ -1,12 +1,11 @@
-# This file is automatically @generated by Poetry 2.1.2 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.8.5 and should not be changed by hand.
```

Comment:
I think the poetry.lock file was committed by mistake

Reply:
good call. must have snuck in while doing renovate items and switching branches. removed